File: code-recipes/java/src/test/java/easyship/GetFeedDocumentRecipeTest.java
package easyship;

import util.RecipeTest;

import java.util.List;

public class GetFeedDocumentRecipeTest extends RecipeTest {

    protected GetFeedDocumentRecipeTest() {
        super(
                new GetFeedDocumentRecipe(),
                List.of(
                    "feeds-getFeed",
                    "feeds-getFeedDocument",
                    ""
                )
        );
    }
}

File: code-recipes/php/test/easyship/GetFeedDocumentRecipeTest.php
<?php

namespace Test\easyship;

use Src\easyship\GetFeedDocumentRecipe;
use Test\util\RecipeTestCase;

class GetFeedDocumentRecipeTest extends RecipeTestCase
{
    protected function setUp(): void
    {
        $this->recipe = new GetFeedDocumentRecipe();
        $this->responses = [
            "feeds-getFeed",
            "feeds-getFeedDocument",
            ""
        ];
    }
}
